packages/react-mutation-mapper/src/component/column/ClinVar.tsx

```diff
@@ -1,41 +1,66 @@
-import { MyVariantInfo } from 'genome-nexus-ts-api-client';
+import { Clinvar, VariantAnnotation } from 'genome-nexus-ts-api-client';
 import { observer } from 'mobx-react';
 import * as React from 'react';
 
-import { defaultSortMethod } from 'cbioportal-utils';
-import ClinVarSummary, {
-    formatClinicalSignificanceText,
-    getRcvCountMap,
-    getRcvData,
-} from '../clinvar/ClinVarSummary';
-import {
-    MyVariantInfoProps,
-    renderMyVariantInfoContent,
-} from './MyVariantInfoHelper';
+import { defaultSortMethod, Mutation, RemoteData } from 'cbioportal-utils';
+import ClinvarSummary from '../clinvar/ClinVarSummary';
+import { getClinvarData } from './ClinVarHelper';
```

Comment:
Should we also rename files as `ClinvarSummary` and `ClinvarHelper` for consistency?